app/code/core/Mage/Core/Model/Translate.php

```diff
@@ -412,7 +424,16 @@ public function translate($args)
 
         //array_unshift($args, $translated);
         //$result = @call_user_func_array('sprintf', $args);
-
+        if (substr_count($translated, '%') != count($args)) {
```

Comment:
Code change. Could you please separate this to an extra PR?

Reply:
Sorry, This doesnt belong in at all.... was just to debug some translation. I'll remove ...